OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoObjectivesLogic.cs

```diff
@@ -64,11 +64,15 @@ void PopulateObjectivesList(MissionObjectives mo, ScrollPanelWidget parent, Cont
 
 			foreach (var o in mo.Objectives.OrderBy(o => o.Type))
 			{
-				var objective = o; // Work around the loop closure issue in older versions of C#
-				var widget = template.Clone();
+				// Work around the loop closure issue in older versions of C#
```

Comment:
Do we even still support the versions that had that issue? Wasn't that in the mono 2.10 days?

Reply:
We don't, and this can be removed.